_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
> `assets/css/main.scss`

Can we talk through a bit the pros and cons of `assets/css/main.scss` versus `/assets/main.scss`? I'm just worried we're over optimizing what is otherwise a simple theme.